test-case/check-alsabat.sh

```diff
@@ -72,11 +79,83 @@ function __upload_wav_file
     done
 }
 
+set_pga_to_unity_gain()
+{
+    tmp=$(amixer controls | grep "$1" | grep Volume)
+    search="name="
+    cname=${tmp#*$search}
+
+    # Get volume min and step to compute value for cset
+    # for 0 dB gain. The amixer line looks like
+    # "| dBscale-min=-50.00dB,step=1.00dB,mute=1"
+    scale=$(amixer cget name="$cname" | grep "dBscale" || true)
+    search="dBscale-min="
+    tmp=${scale#*$search}
+    min_db="${tmp%%dB*}"
+    search="step="
+    tmp=${scale#*$search}
+    step_db="${tmp%%dB*}"
+
+    # Get multiplied by 100 values by removing decimal dot
+    min_x100="${min_db//.}"
+    step_x100="${step_db//.}"
+    val=$(printf %d "$(((-min_x100) / step_x100))")
```

Comment:
Nit, this would be slightly easier to read with some spaces:
```shell
val=$(printf %d "$(( (-min_x100) / step_x100))" )
```
Why do you need the printf at all?